packages/rolldown/tests/parse-ast/parse-ast.test.ts

```diff
@@ -0,0 +1,27 @@
+import { parseAst, parseAstAsync } from 'rolldown/parseAst'
+import { test, expect } from 'vitest'
+
+test('rolldown/parseAst parseSync', async () => {
+  const result = parseAst('test.js', 'console.log("hello")')
+  expect(result.program.type).toBe('Program')
+})
+
+test('rolldown/parseAst parseAstAsync', async () => {
+  const result = await parseAstAsync('test.js', 'console.log("hello")')
+  expect(result.program.type).toBe('Program')
+})
+
+test('rolldown/parseAst parseSync + error', async () => {
+  const result = parseAst('test.js', '\nconso le.log("hello")')
+  try {
+    result.program
```

Comment:
The test might unintentionally pass when the error is missing, so it's better to put unreachable somewhere.

```suggestion
    result.program
    expect.unreachable()
```

Reply:
Thank you.